controllers/storageclaim_controller.go

```diff
@@ -499,6 +500,10 @@ func (r *StorageClaimReconciler) getCephRBDStorageClass(data map[string]string)
 		Provisioner:          csi.GetRBDDriverName(),
 		Parameters:           data,
 	}
+
+	if r.storageClaim.Spec.EncryptionMethod != "" {
+		storageClass.ObjectMeta.Annotations["keyrotation.csiaddons.openshift.io/schedule"] = "@weekly"
```

Comment:
Can you please move the annotation name onto a const at the top of the file, as well as the value? 
Also, why was weekly chosen over other periodicity?

Reply:
> Also, why was weekly chosen over other periodicity?

Values less than weekly would be too soon while values more than it would be too late. Weekly justifies itself as the sweet spot. This decision was also influenced by the fact that ReclaimSpace has its default schedule as weekly.